### DISCUSSION TITLE
Plugin Repo Move Requests from "Issues"

### DISCUSSION THREAD

**Discussion by bcdady** [asdf-community/.github#39] [General]
These Plugins have been nominated for movement to this community Organization in an Issue. Per the [README](https://github.com/asdf-community/.github#how-do-i-move-plugins-here), I'm mentioning them here  in a new discussion to help call attention to them. 

[**Helmsman**](https://github.com/asdf-community/.github/issues/23)

@luisdavim commented on Nov 10, 2021

> Hi,
> 
> I'm one of the maintainers from the [Helmsman](https://github.com/Praqma/helmsman) project, and I've just created an asdf plugin for it, currently in https://github.com/luisdavim/asdf-helmsman, but I'd like to move it to this community org.
> 

[**Teleport**](https://github.com/asdf-community/.github/issues/29)

Merged into [asdf-plugins](https://github.com/asdf-vm/asdf-plugins/pull/570), but not yet to this community

[**Python**](https://github.com/asdf-community/.github/issues/30)

The author/maintainer of the plugin was recently contacted

[**Quarkus**](https://github.com/asdf-community/.github/issues/31)

This too was merged into [asdf-plugins](https://github.com/asdf-vm/asdf-plugins/pull/611), but not yet to this community

[**asdf-pdm**](https://github.com/asdf-community/.github/issues/33)

**Comment by smorimoto** [comment]
I will take care of them in the next couple days...

**Reply by bcdady** [reply]
Sounds great, thanks!!
